### ISSUE TITLE
Feature request: [Feature name]

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-data-tech#2711]
### Show how many permits address is eligible for
- [ ] Have banner message of how many permits customer is eligible for show up once address is populated in `Resident Address` fields.  Similar to this: 
`**Maximum resident decals allowable for purchase**`

![image.png](https://images.zenhubusercontent.com/5cb4c11373d7e234297de5cf/dce84916-2723-420a-9e9c-5c1f38740035)

**Comment by dianamartin** [comment]
Separated this issue from #2467, kept it as the same estimate from the other one.

**Comment by ChrispinP** [comment]
@johnclary @moorerst is this still a need? Customers can check eligibility through the Am I Eligible feature already which displays eligibility for addresses. And im pretty sure that eligibility feature is sticking around so this would require a lot of legwork to do all the automation just for a message that would be in addition to what exists.

Im confident this can be closed out.

**Comment by moorerst** [comment]
@chrispin I agree this can be closed out since the functionality to check eligibility does already exist this was just a nice to have
